.devcontainer/devcontainer.json

```
@@ -33,26 +33,19 @@
         "python.linting.mypyPath": "/usr/local/py-utils/bin/mypy",
         "python.linting.pycodestylePath": "/usr/local/py-utils/bin/pycodestyle",
         "python.linting.pydocstylePath": "/usr/local/py-utils/bin/pydocstyle",
-        "python.linting.pylintPath": "/usr/local/py-utils/bin/pylint",
-        "[python]": {
-          "editor.defaultFormatter": "ms-python.black-formatter"
-        }
+        "python.linting.pylintPath": "/usr/local/py-utils/bin/pylint"
       },
 
       // Add the IDs of extensions you want installed when the container is created.
-      "extensions": [
-        "ms-python.python",
-        "ms-python.vscode-pylance",
-        "ms-python.black-formatter"
-      ]
+      "extensions": ["ms-python.python", "ms-python.vscode-pylance","batisteo.vscode-django"]
     }
   },
 
   // Use 'forwardPorts' to make a list of ports inside the container available locally.
   "forwardPorts": [8000],
 
   // Use 'postCreateCommand' to run commands after the container is created.
-  "postCreateCommand": "python3 -m venv venv && . venv/bin/activate && cd /workspaces/InvenTree && pip install invoke && pip install -r requirements.txt && mkdir -p dev && inv setup-dev && inv migrate && inv server",
+  "postCreateCommand": "python3 -m venv venv && . venv/bin/activate && cd /workspaces/InvenTree && pip install invoke && pip install -r requirements.txt && mkdir -p dev && inv setup-dev && inv update",
```

Suggestion: Nothing from the requirements should be needed to call inv update, if you turn around the order we code safe one action probably:
```suggestion
  "postCreateCommand": "python3 -m venv venv && . venv/bin/activate && cd /workspaces/InvenTree && pip install invoke && inv update && mkdir -p dev && inv setup-dev",
```